blocks/simple-list-block/features/simple-list/default.jsx

```diff
@@ -88,40 +76,41 @@ const SimpleList = (props) => {
 			source: contentService,
 			query: { ...contentConfigValues, feature: "simple-list" },
 			filter: `{
-      content_elements {
-        _id
-        headlines {
-          basic
-        }
-        website_url
-        promo_items {
-          basic {
-            type
-            url
-            resized_params {
-              274x183
-            }
-          }
-          lead_art {
-            promo_items {
-              basic {
-                type
-                url
-                resized_params {
-                  274x183
-                }
-              }
-            }
-            type
-          }
-        }
-        websites {
-          ${arcSite} {
-            website_url
-          }
-        }
-      }
-    }`,
+		content_elements {
+			_id
+			headlines {
+				basic
+			}
+		  	website_url
+		  	promo_items {
+				basic {
+					_id
+					type
+					url
+					auth {
+						${RESIZER_APP_VERSION}
+					}
+			 	}
+				lead_art {
+					promo_items {
+				  		basic {
+							_id
+							type
+							url
+							auth {
+								${RESIZER_APP_VERSION}
+							}
+						}
+					type
+			 	}
+		  	}
```

Comment:
There's a missing closing brace in this block, causing the `websites` to be filtered out. The ANS also doesn't look quite right, `lead_art` should have the same structure as `basic`.

Reply:
Added the missing `}` but not sure about the ANS object here as this is as per before